tools/migrate_gid_to_uid/migrate_gid_to_uid.go

```diff
@@ -222,13 +223,35 @@ func buildMetaIDToUserIDMap(logger log.Logger, config *Config) (map[string]strin
 		} `bson:"private"`
 	}
 	for iter.Next(&result) {
-		if userID := result.UserID; userID == "" {
-			logger.Warn("Missing user id in result from users query")
-		} else if metaID := result.Private.Meta.ID; metaID == "" {
-			logger.WithField("userId", userID).Warn("Missing private meta id in result from users query for user id")
-		} else {
-			metaIDToUserIDMap[metaID] = userID
+		userLogger := logger
+
+		userID := result.UserID
+		if userID == "" {
+			userLogger.Warn("Missing user id in result from users query")
```

Comment:
is it an issue you warn there is no userID but continue and then next statement is setting up logger with nonexistent userID?